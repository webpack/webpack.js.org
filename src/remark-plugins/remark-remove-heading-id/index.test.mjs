import { remark } from 'remark';
import remarkHtml from 'remark-html';
import Plugin from './index.mjs';
import remarkSlug from '../remark-slug/index.mjs';
describe('customize blockquote', () => {
  it('should add id', () => {
    remark()
      .use(remarkSlug)
      .use(Plugin)
      .use(remarkHtml)
      .process(
        `
## hello world

this is me.
`,
        function (err, { value: contents }) {
          expect(err).toBeNull();
          expect(contents).toMatchSnapshot();
        }
      );
  });
});
