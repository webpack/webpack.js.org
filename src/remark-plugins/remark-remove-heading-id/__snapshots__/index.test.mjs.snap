// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`customize blockquote should add id 1`] = `
"<h2><span id=\\"user-content-hello-world\\"></span>hello world</h2>
<p>this is me.</p>
"
`;
